.github/ISSUE_TEMPLATE/vsp-release-plan.md

```diff
@@ -0,0 +1,86 @@
+# Release Plan Template [Draft]
+This Release Plan Template is intended to be used to help ensure your new VSP product or feature is ready for launch and will deliver the desired user outcomes. In Phase 1, you will test your product to learn how it'll actually be used, what problems it might create, and then fix/adjust if necesssary prior to going live. In Phase 2, you will define the launch communications plan and post-launch KPIs. 
+
+You do not need to complete a release plan for a product or feature update that does not:
+
+- Add substantial new functionality to the Platform.
+- Impact user flows through tasks.
+
+For your product or feature, make a copy of this template and check it into products/platform/<product-name>.
+
+## Phase I: User Acceptance Testing
+*Test real-world user scenarios to ensure your product or feature performs as expected.*
+
+### Planning:
+
+- Desired date range or test duration: 
+- Desired number of users: 
+- The group(s) from which you'll recruit test users:
+  - [ ] VFS
+  - [ ] VSP
+  - [ ] DEPO
+- How you'll recuit test users:
+- How you'll give test users access to the product: 
+- What metrics-based "success criteria" will you look at before launch? [use your KPIs to help guide this.]
+- How you'll measure your "success criteria": 
+
+### Results:
+
+- Number of users: 	
+
+- Results for your "success criteria":
+
+- Number of bugs identified / fixed: x/x
+
+- Any UX changes necessary based on the logs, user feedback, or VA challenges? yes/no
+
+  - If yes, what:   
+
+## Phase II: Go Live!
+
+### Launch communications:
```

Comment:
What are we expecting people to do with these checkboxes / bullet points?

How does this overlap with what's already covered in the initiative development checklist?